src/functions.php

```diff
@@ -113,6 +113,119 @@ function stream_for($resource = '', array $options = [])
     throw new \InvalidArgumentException('Invalid resource type: ' . gettype($resource));
 }
 
+
+/**
+ * Return a ServerRequestInterface populated with superglobals :
+ * $_GET
+ * $_POST
+ * $_COOKIE
+ * $_FILES
+ * $_SERVER
+ *
+ * @return ServerRequest
+ */
+function server_request_from_global()
```

Comment:
I think this would be better as a factory method on the ServerRequest class (e.g., `fromGlobals`).